docs/api-android.md

```diff
@@ -1,5 +1,23 @@
 ### Java API Reference (Android)
 
+### API for React Native 0.60 version and above
+
+Since `autolinking` uses `react-native.config.js` to link plugins, constructors are specified in that file. But you can override custom variables to manage the CodePush plugin by placing these values in string resources.
+
+* __Public Key__ - used for bundle verification in Code Signing Feature. Please refer to [Code Signing](setup-android.md#code-signing-setup) section for more details about Code Signing Feature.
```

Comment:
We could call it "the Code Signing Feature":

```suggestion
* __Public Key__ - used for bundle verification in the Code Signing Feature. Please refer to the [Code Signing](setup-android.md#code-signing-setup) section for more details about Code Signing Feature.
```

Reply:
Good catch! Thanks. Updated.